src/SDKs/Network/Management.Network/Generated/SecurityRulesOperations.cs

```diff
@@ -126,7 +126,7 @@ internal SecurityRulesOperations(NetworkManagementClient client)
             {
                 throw new ValidationException(ValidationRules.CannotBeNull, "this.Client.SubscriptionId");
             }
-            string apiVersion = "2018-01-01";
+            string apiVersion = "2018-02-01";
```

Comment:
@murilogr can you confirm if this change was expected.

Reply:
It's expected. If you look at the diff from psSdkJson6 to this branch, we're just changing the version.

For some reason, my first generation used Client.ApiVersion, but the second one reverted to what it was, with "apiVersion".